go-controller/pkg/ovn/master.go

```diff
@@ -155,6 +233,12 @@ func (oc *Controller) StartClusterMaster(masterNodeName string) error {
 		return err
 	}
 
+	err = oc.upgradeOVNTopology(existingNodes)
```

Comment:
If you push another patch, do you mind fixing like 232?

klog.Errorf("Error in initializing/fetching subnets: %v", err)

should be "Error in fetching nodes"